images/win/scripts/Installers/Validate-AzCopy.ps1

```diff
@@ -0,0 +1,7 @@
+if (Get-Command -Name 'azcopy') {
+    Write-Host "azcopy $(azcopy --version) on path"
+}
+else {

```

Comment:
nit: please follow general code style, and place braces and conditions at the same line - `} else {`

Reply:
Done